tough/src/error.rs

```diff
@@ -458,7 +467,16 @@ pub enum Error {
     TargetNotFound { target_url: String },
 
     #[snafu(display("Delegated role not found: {}", name))]
-    DelegateNotFound {
+    DelegateNotFound { name: String },
```

Comment:
This has the same message as `DelegateMissing`. Do both variants need to exist?

Reply:
They have different sources for their errors. I'm not sure if 1 error can have more than 1 source.